src/query/graphite/ts/series_reducer.go

```diff
@@ -28,11 +28,16 @@ type SeriesReducerApproach string
 // The standard set of reducers
 const (
 	SeriesReducerAvg    SeriesReducerApproach = "avg"
-	SeriesReducerSum    SeriesReducerApproach = "total"
+	SeriesReducerSum   SeriesReducerApproach = "sum"
 	SeriesReducerMin    SeriesReducerApproach = "min"
 	SeriesReducerMax    SeriesReducerApproach = "max"
 	SeriesReducerStdDev SeriesReducerApproach = "stddev"
 	SeriesReducerLast   SeriesReducerApproach = "last"
+
+	SeriesReducerAverage    SeriesReducerApproach = "average" // alias for "avg"
+	SeriesReducerTotal    SeriesReducerApproach = "total" // alias for "sum"
+	SeriesReducerCurrent   SeriesReducerApproach = "current" // alias for "last"
+
```

Comment:
No empty endline before end of const block.

Reply:
Fixed.